VEHICLE/GetVehicleDirtLevel.md

```diff
@@ -8,11 +8,18 @@ ns: VEHICLE
 float GET_VEHICLE_DIRT_LEVEL(Vehicle vehicle);
 ```
 
-```
-Dirt level 0..15  
-```
+Gets the dirt level of the passed vehicle 0.0-15.0
```

Comment:
Does it really need a description that repeats the return/required values?

You could also link together with Setter
```
A getter for [`SET_VEHICLE_DIRT_LEVEL`](#_0x79D3B596FE44EE8B).
```